app/assets/javascripts/discourse/app/templates/navigation/filter.hbs

```diff
@@ -4,13 +4,38 @@
       {{d-icon "filter" class="topic-query-filter__icon"}}
       <Input
         class="topic-query-filter__filter-term"
-        @value={{this.queryString}}
+        @value={{this.newQueryString}}
         @enter={{action
           this.discoveryFilter.updateTopicsListQueryParams
-          this.queryString
+          this.newQueryString
         }}
         @type="text"
+        id="queryStringInput"
+        autocomplete="off"
+      />
+      {{! EXPERIMENTAL OUTLET - don't use because it will be removed soon  }}
+      <PluginOutlet
+        @name="below-filter-input"
+        @outletArgs={{hash newQueryString=this.newQueryString}}
```

Comment:
How is this designed to work? It looks like the plugin connector is going to mutate the `newQueryString` argument - is that right?

IMO it would be much cleaner to do something like

```
@outletArgs={{hash updateQueryString=this.updateQueryString queryString=this.queryString}}
```
(where `updateQueryString` is a function)